apps/ledger-live-desktop/src/renderer/modals/CreateLocalManifest/defaultValues.tsx

```diff
@@ -23,6 +23,95 @@ export const DEFAULT_VALUES = {
   platforms: ["ios", "android", "desktop"],
   categories: [""],
   provider: ["evm"],
+  currencies: [
```

Comment:
Isn't there a way to get this programmatically/Dynamically?
I don't want us to have to maintain that list
cc @Justkant